plugin.js

```diff
@@ -25,6 +33,16 @@ function fastifyCookieSetCookie (reply, name, value, options, signer) {
   }
 
   const serialized = cookie.serialize(name, value, opts)
+
+  const bytes = calculateCookieSize(serialized)
+
+  if (bytes >= 4096) {
+    const reachedOrExceeded = bytes > 4096 ? 'exceeded' : 'reached'
```

Comment:
I would make the limit configurable.

Reply:
Done - it's configurable now through the CookieParseOptions as it felt like to be the best place as it's about cookie parsing